### DISCUSSION TITLE
@vue/compiler-sfc] compileTemplate now requires the `id` option

### DISCUSSION THREAD

**Discussion by skiv71** [parcel-bundler/parcel#6222] [General]
How can I remove this warning "properly"?

I say properly, because if I define a component name in my export (which is NOT required), it solves the problem.

But I do NOT want to define the name property, serves no purpose and is NOT the id which @vue/compiler-sfc complains about.

Seems there's some misalignment here, no idea if its parcel related?

**Comment by mischnic** [comment]
Should be fixed in the nightly versions (`yarn add -D parcel@nightly`) by https://github.com/parcel-bundler/parcel/pull/6027
